### PR TITLE
Ia components 0.0.6

### PR DESCRIPTION
**Pull request by iisa**

### CODE REVIEW

File: packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ class DataHydrator extends Component {
     // using bootstrap v3 styling for container for pretty UI
     return (
       <div>
-        <section>
+      <section>
```
Comment (bfalling) [inline packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js:95]: SUGGESTION: Seems like this should still be indented.

File: packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx [hunk 2]
```diff
@@ -49,11 +49,19 @@ export default class BookReaderWrapper extends Component {
...
+      /**
+       * Needed bypass to generate Image URL with scale factor.
+       * We must eliminate sooner than later to allow BookReader full image fetching control
+       * @param {Number} index - page index
+       * @param {Number} reduce - image size scale factor
+       * @param {Number} rotate - degrees of rotation
+       */
       getPageURI: (index, reduce = 1, rotate = 0) => {
-        let uri = originalGetPageURI.call(br, index, reduce, rotate);
+        const brReduce = this.bookreader.reduce || reduce;
```
Comment (bfalling) [inline packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx:61]: QUESTION: So just checking: `this.bookreader.reduce` will now take precedence over any passed-in `reduce`, right?

File: packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx [hunk 3]
```diff
@@ -73,6 +81,7 @@ export default class BookReaderWrapper extends Component {
     } else {
       const br = new BookReader(fullOptions);
       window.br = br;
+      this.bookreader = window.br;
```
Comment (bfalling) [inline packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx:84]: QUESTION: Why is `br` effectively being stored in both `window.br` and `this.bookreader`?

File: packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx [hunk 4]
```diff
@@ -38,7 +39,6 @@ export default class BookReaderWrapper extends Component {
     const originalGetPageURI = BookReader.prototype.getPageURI;
     const defaultOptions = {
       el: `#${this.BookReaderRef.current.id}`,
-      mobileNavFullscreenOnly: true,
```
Comment (bfalling) [inline packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx:41]: QUESTION: Why was this removed?

**Review by bfalling — APPROVED** [review]
Some questions. No blockers.

File: packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx (hunk 2 above)
Comment (iisa) [reply packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx:61]: yes.  this URL string method bypass is NOT the most optimum. in fact we should not even have it at all - but at the moment, we must hold.

We never want the URL to be requested without its image scale, unless you are extremely zoomed, in - that case we handle well enough.

BR 1up mode - sets reducer & reduction factors that allow zooming - and we always want to prefer the calculated image scale at this level. like we did for 2up mode.

File: packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js (hunk 1 above)
Comment (iisa) [reply packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js:95]: yes, unsure why this is sticky. will fix.

File: packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx (hunk 3 above)
Comment (iisa) [reply packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx:84]: window.br is so globally used, we must keep it until we can truly remove the global dependency.

File: packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx (hunk 4 above)
Comment (iisa) [reply packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx:41]: thanks for catching. i've re-added.
did not mean to remove until we move into item-navigator world
